src/Masa.Blazor/Mixins/Selectable/MSelectable.cs

```diff
@@ -48,9 +48,11 @@ public async Task HandleOnChange()
         var input = InternalValue;
         input = !input;
 
-        await ValidateAsync(true, input);
+        ForceValidate(input);
 
-        await SetInternalValueAsync(input);
+        InternalValue = input;
+
+        await Task.CompletedTask;
```

Comment:
Line 58 and 63, This async method lacks 'await' operators and will run synchronously